### PR TITLE
Stabilize test for deterministic_l1 (use smaller variance)

### PR DESCRIPTION
**Pull request by jgasthaus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/distribution/test_distribution_inference.py [hunk 1]
```diff
@@ -347,14 +349,14 @@ def test_deterministic_l1(mu: float, hybridize: bool) -> None:
     class LaplaceFixedVarianceOutput(LaplaceOutput):
         @classmethod
         def domain_map(cls, F, mu, b):
-            b = F.ones_like(b)
-            return mu, b
+            b = 0.1 * F.ones_like(b)
+            return mu.squeeze(axis=-1), b.squeeze(axis=-1)
```
Comment (lostella) [inline test/distribution/test_distribution_inference.py]: These are also required in the `GaussianFixedVarianceOutput` a few lines above, could you add them there too?

**Comment by szha** [comment]
Job PR-58/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-58/1/index.html

**Comment by alexw91** [comment]
# [Codecov](https://codecov.io/gh/awslabs/gluon-ts/pull/58?src=pr&el=h1) Report
> Merging [#58](https://codecov.io/gh/awslabs/gluon-ts/pull/58?src=pr&el=desc) into [master](https://codecov.io/gh/awslabs/gluon-ts/commit/1939cb4769b1b6132081ffcc8f8133fb38cb7ebf?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.


[Impacted file tree graph](https://codecov.io/gh/awslabs/gluon-ts/pull/58?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master      #58   +/-   ##
=======================================
  Coverage   77.81%   77.81%           
=======================================
  Files         108      108           
  Lines        6189     6189           
=======================================
  Hits         4816     4816           
  Misses       1373     1373
```

**Comment by szha** [comment]
Job PR-58/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-58/2/index.html

**Comment by szha** [comment]
Job PR-58/3 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-58/3/index.html

**Comment by szha** [comment]
Job PR-58/4 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-58/4/index.html

**Comment by szha** [comment]
Job PR-58/5 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-58/5/index.html

**Pull request merged**